### DISCUSSION TITLE
Routing "Etiquette"

### DISCUSSION THREAD

**Discussion by K0JEG** [meshtastic/Discussions#47] [General]
I saw [this video](https://youtu.be/htjwtnjQkkE?si=dQuK_Dg5bocyqYyG) last night and it got me wondering what motivated it. My interpretation of how Meshtastic should work is that it is very local in nature, covering a  neighborhood or a square miles. If you want to get more reach, run an MQTT server and build out infrastructure to support it. Am I wrong in this assumption?

It seems to me while long distance hop experiments are interesting, ultimately long haul Meshtastic nodes are unnecessary and will only lead to frustration from people who wish to manage the network instead of adapting to it. I thought the whole point was to have an ad-hock network capability with some assistance from permanent nodes. Yes, most people should be setting their radios to client, which I believe is the default and mentioned in the documentation. But attempting to optimize for wide area networks using Meshtastic doesn't seem like a good use of the channel. 

Perhaps bridging local areas with more traditional LoRA or other 900 MHz point to point links would be a better solution down the road? Note I'm in the Western US in a rural area that doesn't have much Meshtastic penetration yet. I've been considering asking for some tower space at my ham radio club's repeater facilities but then I wonder how useful it would be? I'm still leaning more towards MQTT, even with 9600 baud packet or other radio modems off the 900MHz band (or just Internet connections until Meshtastic gains some traction in the area).

thoughts?

**Comment by ikearat** [comment]
I enjoyed watching the video. I came away with thoughts about how my local area is setup. Other than my own node, there is a single other node in view and it is setup as Router. I have wondered why this node was configured like this and after a zig-zag bike ride a few days ago and using the RSSI "signal strength" I think the node is about 1/2 way up a local commercial tower. The site is a "hosted" site with many different customers. About the user's choice of "Router". I think they are using the Client-mute settings on the nodes using this router as on occasion I see nodes with no name and no location or telemetry. Why this, no idea but there are other nodes besides my own and the Router. I am considering setting my node to Client-mute or not. Does my ground level node which is likely only reaching a short section of the local farm to market road contributing to the mesh or am I loading the router with unusable traffic which as I see comes and goes like it's from mobile nodes. Another thread here on the Github Discussions is requesting a choice in the user interface (whatever device you are using) that "handles" these kinds of decisions for the user... a "Experienced User" setting that when not selected, chooses and configures your node to be the most beneficial in consideration of the preexisting mesh when you come online.

**Comment by mariotti** [comment]
Hoi!
I think it is an interesting video. It shows some scenarios and suggestions. After the video I started to play a bit with client, client_mute, but me myself I am far from a network :) . In practice it depends a lot on the algorithm.

But I fully agree:
```
who wish to manage the network instead of adapting to it
```

I think the video is more informative then practical.

Just to tell people, do not get your first 2 device and set a ROUTER and a CLIENT.
 Or even just 1 and set it as ROUTER, because it helps everybody.

Or get this thread starting :) 

To be honest I just started because I got interested in devices like:
t-watch, t-deck
To find out that there is a lot of more background on "sensor" devices.
I am not sure we are going to use the same netiquette in a short time.
(in the mean time I got more static devices ;) )

Cheers
F
